pandas/tests/io/excel/test_readers.py

```diff
@@ -398,20 +399,25 @@ def test_excel_stop_iterator(self, read_ext):
         expected = DataFrame([["aaaa", "bbbbb"]], columns=["Test", "Test1"])
         tm.assert_frame_equal(parsed, expected)
 
-    def test_excel_cell_error_na(self, request, read_ext):
-        if read_ext == ".xlsb":
+    def test_excel_cell_error_na(self, request, engine, read_ext):
+        if engine == "pyxlsb":
             request.node.add_marker(
                 pytest.mark.xfail(
                     reason="Sheets containing datetimes not supported by pyxlsb"
                 )
             )
 
+        if engine == "calamine" and read_ext == ".ods":
+            request.node.add_marker(
+                pytest.mark.xfail(reason="Calamine can't extract error from ods files")
+            )
+
```

Comment:
Does calamine raise an exception here? If so, can you test for that using `pytest.raises` instead. I think we should be doing this for unsupported features across the readers instead of xfailing.

Reply:
Not exception. Value of cell, because `office:value` is present (`<table:table-cell office:value-type="float" office:value="0" table:formula="of:=#N/A" table:style-name="ce1"><text:p>#N/A</text:p></table:table-cell>`). I created issue for this.
* https://github.com/tafia/calamine/issues/355